.taskcluster.yml

```diff
@@ -202,7 +202,7 @@ tasks:
                         path: /build/android-components/parameters.yml
                         expires: ${expires_in}
               in:
-                - $if: 'tasks_for == "github-release"'
+                - $if: 'tasks_for == "github-release" && event["action"] in ["published"]'
```

Comment:
Nit: Change to:
```suggestion
                - $if: 'tasks_for == "github-release" && event["action"] == "published"'
```

(I don't know how yaml parses lists containing a single string. If it's like a python tuple, it may be a bug

Reply:
Oh, good catch! I copy-pasted from above, oups 😊